src/main/java/com/sparta/twitNation/dto/post/resp/PostCreateRespDto.java

```diff
@@ -1,4 +1,10 @@
 package com.sparta.twitNation.dto.post.resp;
 
-public class PostCreateRespDto {
+import com.sparta.twitNation.domain.post.Post;
+
+public record PostCreateRespDto(Long postId) {
+
+    public PostCreateRespDto (Post post){
+        this(post.getId());
```

Comment:
가능하면 띄어쓰기도 맞춰주는걸로..!

```suggestion
    public PostCreateRespDto(Post post) {
        this(post.getId());
```

Reply:
네! 수정하겠습니다

Reply:
확인했습니당